### ISSUE TITLE
Report various stats as rates rather than aggregates

### ISSUE THREAD

**Issue by davidtgoldblatt** [jemalloc/jemalloc#1380]
For some aggregate stats, we're often more interested in rates than absolute counts. Reporting the aggregates makes it hard to compare stats across processes with different uptimes.

Stats that I think are easier to understand as rates instead:
- nmalloc, ndalloc, nrequests
- nfills, nflushes, nreslabs
- n_lock_ops, n_waiting, n_spin_acq, n_owner_switch, total_wait_ns

Dividing by uptime turns these into easier-to-interpret outputs; whether or not total_wait_ns is too high is hard to parse visually, whereas "we spend X% of our runtime waiting on this malloc mutex" is something that's easy to understand. "This process has done N mallocs" doesn't tell us if it's an allocation-heavy process, "this process does N mallocs per second" does.

(I don't suggest any changes for the machine-friendly APIs, just the human-readable output).

**Comment by interwq** [comment]
FYI that Alexander Zinoviev is working on this.

**Comment by interwq** [comment]
This is added in the dev branch. Thanks Alexander for working on this!
